controllers/os-coreos/pkg/coreos/add.go

```diff
@@ -21,12 +21,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
-// Type is the type of OperatingSystemConfigs the coreos actuator / predicate are built for.
-const Type = "coreos"
-
 var (
 	// DefaultAddOptions are the default controller.Options for AddToManager.
 	DefaultAddOptions = AddOptions{}
+	// Types is the type of OperatingSystemConfigs the coreos actuator / predicate are built for.
+	Types = []string{"coreos"}
```

Comment:
```suggestion
	Types = []string{"coreos", "flatcar"}
```

Reply:
(FYI: What @ialidzhikov suggests should be done via https://github.com/gardener/gardener-extensions/blob/master/controllers/os-coreos/charts/os-coreos/doc.go#L15 and `go generate`)

Reply:
I would create other PR´s which can then take care of further simplifications on the controller side. 
I´m fine merging that now.

Reply:
Fair enough. 👍